### ISSUE TITLE
Number of player slots...

### ISSUE THREAD

**Issue by morgoth0** [EpochModTeam/Epoch#915]
So, as we all know, there are actually 201 player slots defined in mission.sqm (Item3 - Item203) which gives an irrelevant (but annoying) warning in the RPT file. Want me to fix them in the MPMissions source?

**Comment by vbawol** [comment]
Yes please, even though I used a script to make these I must have missed something.

**Comment by morgoth0** [comment]
I have made a pull request removing the extra player slots from all mission.sqm files. As an aside, the script you used would not result in the RPT file error

`15:01:02 Mission epoch.ChernarusRedux: Number of roles (201) is different from 'description.ext::Header::maxPlayer' (200)
`
on a server which had not added anything to the mission file. 

The reason for that is, in the cases where the last player slot was Item203, the "items=xxx" line was in fact "items=203", not the correct "items=204" so only if you added an item and set the items line correctly would the RPT show the extra slot.

This change should be highlighted in the ChangeLog for folks who have modded their mission.sqms and not removed that last item in case they actually add their custom items to the new mission.sqm on an update (personally I just check to make sure that nothing's changed and copy over my new one... only time I had to manually alter the Epoch one was after the Eden release and after the increase to 200 slots...)

Cheers,
Grahame.
